### DISCUSSION TITLE
Domotizzare VMC Mitsubishi

### DISCUSSION THREAD

**Discussion by simarelli** [indomus/forum#2893] [Nuove idee, proposte, progetti e guide]
Buona sera a tutti. Sono nuovo del forum e volevo sottoporre al gruppo una soluzione per domotizzare una macchina di ventilazione meccanica controllata della Mitsubishi In pratica si tratta di una macchina che estrae aria viziata dalla stanza e ne immette di nuova dall'esterno recuperando il calore tra uscita e ingresso. Di fatto è costituita da due ventilatori che lavorano sincronicamente. A differenza di altri apparecchi più sofisticati non ha a bordo alcun tipo di elettronica e viene comandata manualmente attraverso un interruttore acceso/spento e da un selettore di 2 velocità tra cui si può scegliere. Di base vorrei poter rendere smart questo tipo di interruttore acceso/spento e con la possibilità di selezionare una delle due velocità magari potendo effettuare una programmazione temporale dei cambi di velocità (ad esempio velocità elevata durante il giorno, e velocità minima durante la notte). Successivamente mi piacerebbe poter comandare automaticamente la variazione di velocità in base ai dati rilevati da un sensore di umidità, o di co2. Vorrei chiedere ai più esperti del gruppo quali componenti utilizzare e gli schemi di collegamento. Ringrazio infinitamente per chi potesse aiutarmi

**Comment by Bergoglio** [comment]
Domande per capire meglio e, spero, aiutarti :

1. che modello di macchinario è ?
2. questi interruttori (acceso/spento e selettore di velocità) sono installati sulla macchina o sono in un quadro comandi separato collegato alla macchina mediante comunissimi fili elettrici ?

Questo tanto per definire l'hardware, altrimenti poi non si sa come comandarla.

**Comment by simarelli** [comment]
Ciao. Si tratta di una macchina di ventilazione meccanica controllata che
finalmente è formata da due ventilatori che estraggono e immettono
contemporaneamente l"aria da e verso l"esterno! Il modello è il Mitsubishi
vl 100. I controlli sono esterni alla macchina. In pratica ci sarebbe un
interruttore acceso/spento e un altro selettore che permette di scegliere
la velocità 1 o la velocità 2 inseriti in una scatoletta 503. Allego
manuale di installazione della macchina.

Il Ven 8 Mar 2024, 23:21 Bergoglio ***@***.***> ha scritto:

**Comment by Bergoglio** [comment]
Non vedo qui il manuale ma l'ho cercato in rete.
Lo schema elettrico mi pare molto semplice :
1) la fase dovrebbe arrivare all'interruttore di acceso/spento e da lì andare al morsetto COM sulla morsettiera della macchina
2) il neutro invece dovrebbe essere al polo comune di un semplice deviatore (che svolge così funzione di scelta velocità) e le 2 uscite del deviatore andare ai morsetti HIGH/LOW sulla morsettiera della macchina.
Controlla che sia veramente così sulla tua unità per evitare casini.

Puoi sostituire l'interruttore ed il selettore con dei relè Shelly, dei Sonoff, delle "schede bianche" cinesi da programmare con ESP Home etc. etc. etc.
L'unica mancanza degli Shelly è l'eventuale assenza di sicurezza fisica (puoi contare solo sul controllo software per non attivare in contemporanea velocità High e Low).
